devtools/conda-envs/test_env.yaml

```diff
@@ -32,6 +32,6 @@ dependencies:
   - gromacs >=2021
   - lammps
   - mbuild
-  - foyer>=0.8.0
+  - foyer>=0.8.1
```

Comment:
@umesh-timalsina @justinGilmer I looked at the [changelog](https://github.com/mosdef-hub/foyer/releases) and deduced this was the minimum version for this patch - let me know if that's not the case!

Reply:
It looks like the graph conversion support was added in [`0.9.0`](https://github.com/mosdef-hub/foyer/releases/tag/0.9.0) for foyer and there's was a hot bugfix patch release as `0.9.1`.

Reply:
Okay, I'll bump it up to 0.9.1